migrations/0001_add_tables.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS parcel_status (
+  id serial PRIMARY KEY,
+  status_value TEXT
+);
+
+CREATE TABLE IF NOT EXISTS parcel (
+  id serial PRIMARY KEY,
+  user_id int,
```

Comment:
use not null constant

Reply:
for user_id field?

Reply:
added not null constant for user_id and carrier_id